src/rosdiscover/recover/analyzer.py

```diff
@@ -8,6 +8,7 @@
 import typing as t
 
 import attr
+from functools import cached_property
```

Comment:
bit of a nitpick: this should go into the first group of imports (standard library)

Reply:
fixed